lib/faraday/adapter/excon.rb

```diff
@@ -15,6 +15,9 @@ def call(env)
           opts[:client_key]  = ssl[:client_key]  if ssl[:client_key]
           opts[:certificate] = ssl[:certificate] if ssl[:certificate]
           opts[:private_key] = ssl[:private_key] if ssl[:private_key]
+          opts[:ssl_version] = ssl[:ssl_version] if ssl[:ssl_version]
```

Comment:
There's probably a typo here, should be `ssl[:version]`.

Reply:
Woops thanks!